packages/portal/swagger-ui-web-component/src/element.js

```diff
@@ -172,6 +190,24 @@ export class SwaggerUIElement extends HTMLElement {
       this.dispatchEvent(new CustomEvent('clicked-register', { bubbles: true }))
     }
 
+    const defaultTheme = {
+      hasSidebar: this.#hasSidebar,
+      applicationRegistrationEnabled: this.#applicationRegistrationEnabled,
+      currentVersion: { version: this.#currentVersion },
+      onViewSpecClick,
+      onRegisterClick,
+    }
+
+    const mergedThemeObject = { ...defaultTheme, ...this.#themeOverrides }
+
+    const plugins = [SwaggerUI.plugins.DownloadUrl]
+
+    if (this.#customSwaggerTheme) {
```

Comment:
I could use spread operator to add it to the array, but I think this is more readable.